### ISSUE TITLE
no stats, no service nor host details

### ISSUE THREAD

**Issue by MikeW83** [centreon/centreon#7151]
Hello!

I have tried to configure a remot server described at https://documentation.centreon.com/docs/centreon/en/latest/administration_guide/poller/install_remote_server_iso.html and it's not working.

I see the host and the services at the remot server via web ui. BUT when I click on a service I get this error message:
![grafik](https://user-images.githubusercontent.com/35097389/51462684-87ddbd00-1d61-11e9-9bed-31d200b28969.png)

- LDAP import not possible
I cannot import any user from LDAP. I select the user from the search results and nothing happens.

- Graphs not correct
I had this issue a few days ago with the centrel server. Now I have the same on my remote server. On central server the graph for incoming and outgoing traffic is correct. But on remote server I don't the the traffic for incoming and outgoing like in the demo.

What I'm doing wrong? Can anybody help me to check the configuration?
Thank you!

Best regards, Michael

**Comment by victorvassilev** [comment]
Hello Michael,

Could you please let us know what was the problem with your central server that you resolved?

I'd assume that you set the remote up before fixing that specific problem and the configuration needs to be exported to the remote now as well. 

You can try to go to the poller that is corresponding to the remote server and export configuration for it, just to be sure.

Let me know if this doesnt work and please include brief explanation what you changed to fix the central so that we can determine if it is relative to the remote server problem.

**Comment by MikeW83** [comment]
The problem with the graphs has been already solved. I had forgotten to add the template to the service. But that is not the problem. The main problem is, that when I connect to the remote server I have the problems described above.
The setup has been done like it's described in the documentation.

**Comment by victorvassilev** [comment]
After adding the template to the service, did you do export configuration of the remote server connected poller? The configuration from the central needs to be exported/imported to/into remote to apply. Please try that and let me know if it doesnt work so we can debug further.

**Comment by MikeW83** [comment]
> 
> 
> After adding the template to the service, did you do export configuration of the remote server connected poller? The configuration from the central needs to be exported/imported to/into remote to apply. Please try that and let me know if it doesnt work so we can debug further.

Yes, I have done everything like with the other poller. The only difference is, that I have now a Remote Server and not a poller. But the stats are not the main problem. When you look again at my problem description, you will see, I have other problems which have more priority.
We can correct the part with the graph at the end.

At first I need a solution for the other things. The main problem is, that at the Remote Server I don't have any details. No service nor host details or any other stats. I can only view these things at the Central Server.

Sorry if I haven't exactly explained my problem.

**Comment by MikeW83** [comment]
I found another strange thing:
![grafik](https://user-images.githubusercontent.com/35097389/51670848-e3a97f80-1fc7-11e9-9ff7-909ef8349334.png)
When I want to check the broker status I don't see any data. AND the name above for the broker is wrong. It is NOT 'Central' ... Do I need to do a complete new installation of the Remote Server? :-(

**Comment by lpinsivy** [comment]
Hi,

I support the centcore process is not running on your Remote Server.
Can you start it:

`# systemctl restart centcore`

Regards,

**Comment by snakeeyes5135** [comment]
Hello,
i have the same problem 
This error appears on my RMS and i don't know why 

For LDAP same issue the link is good and i can see all of my LDAP users but import doesn't work 
When the user log in for the first time her name his add to users list

**Comment by cgagnaire** [comment]
Hi,
Do you still have this kind of problem?
